### PR TITLE
opmon protobuf microservice

### PR DESCRIPTION
**Pull request by mroda88**

### CODE REVIEW

**Comment by jcpunk** [comment]
Should this PR remove the `telegraf` instance or were we looking to run that in parallel for a while?

**Comment by mroda88** [comment]
We will be running in parallel until all the applications in the framework will use the new system. This is the full support so that we can test immediately once we develop the applications.

**Comment by jcpunk** [comment]
You'll also need to add:
```yaml
  - path: opmo-protobuf-dbwriter
```

to `dunedaq_services/opmon/deployment.yml`

**Comment by mroda88** [comment]
Done.

We need to understand if the username and passwords to write on influx are necessary.

File: dune_daq_services/opmon/opmon-protobuf-dbwriter/kustomization.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    patch: |-
+      - op: replace
+        path: /spec/template/spec/containers/0/image
+        value: {{ dunedaq.unified_microservices.image }}:{{ dunedaq.unified_microservices.tag }}
+      - op: replace
+        path: /spec/template/spec/containers/0/env/1/value
+        value: {{ dunedaq.kafka.bootstrap_brokers[0] }}
+      - op: replace
+        path: /spec/template/spec/containers/0/env/2/value
+        value: {{ OPMON_telegraf.influxdb_username }}
```
Comment (jcpunk) [inline dune_daq_services/opmon/opmon-protobuf-dbwriter/kustomization.yml]: I think `DUNE_opmon.influxdb_rw_username` and `OPMON_opmon.influxdb_rw_password` are the right values here.

File: dune_daq_services/opmon/opmon-protobuf-dbwriter/kustomization.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        value: {{ dunedaq.unified_microservices.image }}:{{ dunedaq.unified_microservices.tag }}
+      - op: replace
+        path: /spec/template/spec/containers/0/env/1/value
+        value: {{ dunedaq.kafka.bootstrap_brokers[0] }}
+      - op: replace
+        path: /spec/template/spec/containers/0/env/2/value
+        value: {{ DUNE_opmon.influxdb_rw_username }}
+      - op: replace
+        path: /spec/template/spec/containers/0/env/3/value
+        value: {{ OPMON_opmon.influxdb_rw_password }}
```
Comment (jcpunk) [inline dune_daq_services/opmon/opmon-protobuf-dbwriter/kustomization.yml]: I think you mean `{{ DUNE_opmon.influxdb_rw_password }}`

Comment (mroda88) [reply dune_daq_services/opmon/opmon-protobuf-dbwriter/kustomization.yml]: Fixed

Comment (jcpunk) [reply dune_daq_services/opmon/opmon-protobuf-dbwriter/kustomization.yml]: I wouldn't have seen it except for the render error

**Comment by jcpunk** [comment]
Closed in favor of https://github.com/DUNE-DAQ/daq-kube/pull/182

**Pull request closed without merging**